main/test/ca/uwaterloo/flix/library/TestFile.flix

```diff
@@ -229,6 +229,13 @@ namespace TestFile {
             case _ => false
         }
 
+    @test
+    def testReadBytesWith03(): Bool & Impure =
+        match readBytesWith({offset = 10i64, count = 4}, "./LICENSE.md") {
```

Comment:
There should also be a test with offset 0 and one with count 0 (maybe also some with negative numbers)